src/main/kotlin/uk/gov/gdx/datashare/resource/EventPlatform.kt

```diff
@@ -36,8 +36,8 @@ class EventPlatform(
     ]
   )
   suspend fun getEvents(
-    @Schema(description = "Event Types required, if none supplied it will be the allowed types for this client", required = false)
-    @RequestParam(name = "eventType") eventTypes: List<EventType> = listOf(),
+    @Schema(description = "Event Types required, if none supplied it will be the allowed types for this client", required = false, allowableValues = [ "DEATH_NOTIFICATION", "LIFE_EVENT"])
```

Comment:
This says required, and then `required = false`

Reply:
Done